spec/system/task_spec.rb

```diff
@@ -59,22 +60,20 @@
 
   describe 'Task編集' do
     context '正常系' do
-      xit 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
+      let(:task) { create(:task) }
+      let(:task_done) { create(:task, :task_completed_yesterday) }
```

Comment:
trait名にtaskはわざわざ付けなくても良いと思います。ステータス名と合わせて`done`でも良いと思います。

`yesterday`であることも、他の期間を表す`trait`がない場合は省いてしまっても良いかと思います。
```suggestion
      let(:task_done) { create(:task, :completed) }
```

Reply:
https://github.com/daigomiyoshi/rspec_app_exam/pull/1/commits/659c7e6a4e8e09f4c9157f1ecc434fbf21381b15 にて修正いたしました！